WooCommerce/Classes/ViewRelated/Orders/Payment Methods/Cash/CashPaymentTenderView.swift

```diff
@@ -0,0 +1,131 @@
+                        Text(viewModel.changeDue)
+                            .font(.system(size: Layout.amountFontSize, weight: .bold))
+                            .foregroundColor(Color(.label))
+                            .opacity(viewModel.hasChangeDue ? 1: 0.18)
+                    }
+                    .padding(Layout.sectionPadding)
+
+                    Toggle(Localization.addNoteToggleTitle, isOn: $viewModel.addNote)
+                        .padding(Layout.togglePadding)
+                }
+            }
+
+            VStack(spacing: 0) {
+                Divider()
+                    .frame(height: 1)
+                    .foregroundColor(Color(.separator))
+
+                Button(Localization.markOrderAsCompleteButtonTitle) {
+                    viewModel.onMarkOrderAsCompleteButtonTapped()
+                    dismiss()
+                }
+                    .buttonStyle(PrimaryButtonStyle())
+                    .disabled(!viewModel.tenderButtonIsEnabled)
+                    .padding(insets: Layout.buttonPadding)
+            }
+        }
+        .background(Color(.systemBackground))
+        .navigationTitle(String(format: Localization.navigationBarTitle, viewModel.formattedTotal))
+        .navigationBarTitleDisplayMode(.inline)
+        .navigationBarBackButtonHidden()
+        .toolbar {
+            ToolbarItem(placement: .cancellationAction) {
+                Button(action: {
+                    withAnimation {
+                        dismiss()
+                    }
+                }, label: {
+                    Text(Localization.cancelButtonTitle)
+                })
+            }
+        }
+    }
+}
+
+extension CashPaymentTenderView {
+    enum Layout {
+        static let amountFontSize: CGFloat = 48
+        static let verticalSpacing: CGFloat = 24
+        static let dividerHorizontalPadding: CGFloat = 16
+        static let sectionPadding: EdgeInsets = .init(top: 24, leading: 16, bottom: 8, trailing: 16)
+        static let sectionVerticalSpacing: CGFloat = 4
+        static let togglePadding: EdgeInsets = .init(top: 8, leading: 16, bottom: 16, trailing: 16)
+        static let buttonPadding: EdgeInsets = .init(top: 16, leading: 16, bottom: 16, trailing: 16)
+    }
+
+    enum Localization {
+        static let customerPaidTitle = NSLocalizedString("cashPaymentTenderView.customerPaid",
+                                                         value: "Cash received",
+                                                         comment: "Title for the amount the customer paid.")
+        static let changeDueTitle = NSLocalizedString("cashPaymentTenderView.changeDue",
+                                                      value: "Change due",
+                                                      comment: "Title for the change due text.")
+        static let addNoteToggleTitle = NSLocalizedString(
+            "cashPaymentTenderView.addNoteToggle.title",
```

Comment:
String is in new line:
NSLocalizedString(
            "cashPaymentTenderView.addNoteToggle.title"
instead of 
NSLocalizedString("cashPaymentTenderView.addNoteToggle.title"